src/main/python/powerFactory2json/pf2json.py

```diff
@@ -20,17 +19,26 @@
 # Script #
 ##########
 
+def name_without_preamble(full_name):
+    """
+    Remove name pollution by omitting uniform file path preamble
+    """
+    flags = ["\\Network Data.IntPrjfolder\\", "\\Equipment Type Library.IntPrjfolder\\"]
+    for flag in flags:
+        if flag in full_name:
+            return full_name.split(flag)[-1]
+    return full_name
```

Comment:
Oh, wow. :exploding_head: 
I can just recommend to have a look into Regex. Makes things easier, after you got the concept. ^^